apps/voting/contracts/Voting.sol

```diff
@@ -73,6 +96,9 @@ contract Voting is IForwarder, AragonApp {
     event ChangeMinQuorum(uint64 minAcceptQuorumPct);
     event ChangeVoteTime(uint64 voteTime);
     event ChangeObjectionPhaseTime(uint64 objectionPhaseTime);
+    event SetDelegate(address indexed voter, address indexed delegate);
+    event ResetDelegate(address indexed voter, address indexed delegate);
+    event CastVoteAsDelegate(uint256 indexed voteId, address indexed delegate, address indexed voter, bool supports, uint256 stake);
```

Comment:
Maybe let's optimize `CastVoteAsDelegate` event and cut off `supports` and  `stake` fields? it's additional 2*256 units of gas! Or maybe get rid of indexed params?

Reply:
Addressed in 32a3dd239c42b642c2cbba3065bbf66e2f4a61b0. 

(comment TBA)